climate_categories/data/PRIMAP_entity.yaml

```diff
@@ -0,0 +1,1627 @@
+name: PRIMAP_entity
+title: PRIMAP2 entities
+comment: Entities commonly used in PRIMAP2
+references: Many emissions entities are derived from openscm_units (https://github.com/openscm/openscm-units),
+  other entities are collected for PRIMAP2.
+institution: PIK
+last_update: 2021-03-09
+version: 2
+total_sum: False
+categories:
+  C:
```

Comment:
I think a field "formatted_title" with the formula in e.g. latex code would also be nice to automatically generate properly formatted entity names in plots etc.

Reply:
The long name without underscore is now simply the title. The things with underscores are just additional codes.

The formula in latex code is a fancy idea, is it common enough to add as a new standard attribute, with an automatic fallback to the unformatted title? That would make it easy to use, but optional when defining categories.

Reply:
I put this discussion/feature request into a separate issue.